HOWTOMOVE.md

```diff
@@ -0,0 +1,30 @@
+# How To Move a Utility Pkg from Other Kubernetes Repos
+
+It has 2 steps to move a pkg from other Kubernetes repos to `k8s.io/utils` repo:
+- copy the pkg to `k8s.io/utils` repo
+- uddate the import paths and `vendor/` in the repos that refer this pkg
+
+## Copy the Pkg to `k8s.io/utils` Repo
+
+Copying should preserve all the git history associated with it.
+[Here](http://gbayer.com/development/moving-files-from-one-git-repository-to-another-preserving-history/) is a working approach.
```

Comment:
I have tried to tweak `git filter-branch` a bit, but I found the way mentioned above is the actually the easiest working approach.
If someone find a better way to do it, we can update this doc :)